src/common/imageio_rawspeed.cc

```diff
@@ -120,7 +120,7 @@ dt_imageio_retval_t dt_imageio_open_rawspeed(dt_image_t *img, const char *filena
 
 #ifdef __WIN32__
   const size_t len = strlen(filename) + 1;
-  wchar_t filen[len];
+  wchar_t filen[MAX_PATH];
```

Comment:
I know it sounds strange, but in WIN32 its called MAX_PATH. As far as I know there is no cross-platform equivalent: [MAX_PATH](https://msdn.microsoft.com/en-us/library/windows/desktop/aa365247(v=vs.85).aspx#maxpath)